src/Cmts.ml

```diff
@@ -207,7 +207,7 @@ end = struct
   let empty : t = (empty_start, empty_end)
 
   let invariant (smap, emap) =
-    assert (List.equal Poly.equal (Map.to_alist smap) (Map.to_alist emap))
+    assert (Poly.equal (Map.to_alist smap) (Map.to_alist emap))
```

Comment:
Why not `Map.equal Poly.equal smap emap` ?

Reply:
`smap` and `emap` don't have the same type.

Reply:
I see.